### ISSUE TITLE
How do I add Pages after project creation

### ISSUE THREAD

**Issue by andyphilpotts** [Microsoft/WindowsTemplateStudio#376]
I want to add pages to an existing, Wizard generated project, I can find no "add item... TemplatedPage" kind of option.

It seems I have to know ahead of time all the pages I want to add, but that is not very agile.

I am sure I am missing something obvious!

**Comment by crutkas** [comment]
We are adding right click add.  Had to ship and it is a hard thing to nail due to the complexity added of altering a project in an unknown state. Generating a file->new is mich easier as we control everything.  

We have a few ideas pending.  To unblock, I would start a new project and copy in existing pages. Less than ideal but nonblocking.

**Comment by andyphilpotts** [comment]
Thanks, thankfully, the template I used is lightweight enough, it is easy to handle new pages etc. manually, I thought I was just being dense, or there was something broken.

Kudos on keeping the code clean and simple

**Comment by mrlacey** [comment]
Closing as a duplicate of #267 

Note that taking time to plan all that you'll want to include in an app before you start is also a good thing 😉
